src/client/src/apps/MainRoute/widgets/spotTile/components/TileSwitch.tsx

```diff
@@ -47,18 +45,10 @@ const TileSwitch: React.FC<Props> = ({
     isRfqStateNone,
   } = getConstsFromRfqState(spotTileData.rfqState)
 
-  const spotTileDataWithNotional =
```

Comment:
this logic has been moved to a selector (see `widgets/spotTile/selectors.ts`). The problem of having this data modification in the view is that we would need to duplicate this code if any other view needs this data slice too